### PR TITLE
Update assign-licenses-to-users.md

### PR DESCRIPTION
**Pull request by msbemba**
Added steps to assign license to guest users (Mostly required for powerbi liceneses)

### CODE REVIEW

**Comment by msbemba** [comment]
Issue# https://github.com/MicrosoftDocs/OfficeDocs-O365ITPro/issues/975

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Office365-Admin/manage/assign-licenses-to-users.md [hunk 1]
```diff
@@ -220,6 +220,16 @@ If you have more than one subscription, and you have users who already have a li
 
 ::: moniker-end
 
+## Assign licenses to  Guest users
+
+1. In the admin center, go to the **Users** \> <a href="https://go.microsoft.com/fwlink/p/?linkid=850628" target="_blank">Active users</a> page.
+
+2. Select the boxes next to the names of the Guest user you want to add licenses for.
+
+3. Click on the **Edit Product Licenses**
```
Comment (JohanFreelancer9) [inline Office365-Admin/manage/assign-licenses-to-users.md]:
```suggestion
3. Select **Edit Product Licenses**.
```

File: Office365-Admin/manage/assign-licenses-to-users.md [hunk 2]
```diff
@@ -220,6 +220,16 @@ If you have more than one subscription, and you have users who already have a li
 
 ::: moniker-end
 
+## Assign licenses to  Guest users
+
+1. In the admin center, go to the **Users** \> <a href="https://go.microsoft.com/fwlink/p/?linkid=850628" target="_blank">Active users</a> page.
+
+2. Select the boxes next to the names of the Guest user you want to add licenses for.
```
Comment (JohanFreelancer9) [inline Office365-Admin/manage/assign-licenses-to-users.md]:
```suggestion
2. Select the boxes next to the names of the Guest users you want to add licenses for.
```

File: Office365-Admin/manage/assign-licenses-to-users.md [hunk 3]
```diff
@@ -220,6 +220,16 @@ If you have more than one subscription, and you have users who already have a li
...
+
+3. Click on the **Edit Product Licenses** 
+
+4. Assign the desired license.
```
Comment (JohanFreelancer9) [inline Office365-Admin/manage/assign-licenses-to-users.md]:
```suggestion
4. Assign the required license.
```

**Comment by msbemba** [comment]
> As mentioned in the OP - You cannot assign licenses to Guest Users in the "Active users" page, whether using the new Microsoft 365 admin center or old. The selection for "Edit Product Licenses" just does not exist in the UI.

Thank you, I don't see the option in the new Admin center or old admin center anymore. This needs to be done via Azure, Ill update you shortly once I draft the steps.

**Pull request closed without merging**